src/Context/LoginContext.php

```diff
@@ -245,22 +266,48 @@ public function stepPasswordForEmailShouldBe($id, $password)
      * @return Member
      */
     protected function generateMemberWithPermission($email, $password, $permCode)
+    {
+        $group = $this->generateGroupWithPermission($permCode);
+
+        return $this->generateMember($email, $password, $group, $permCode);
+    }
+
+    /**
+     * Get or generate a group with the given permission code
+     *
+     * @param string $groupName
```

Comment:
```suggestion
     * @param string $permCode
```
This is still the permission code.